src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentSaleRequest.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentSaleRequest
+    {
+        public TrustlyPaymentSaleRequest(Msisdn msisdn)
+        {
+            Transaction = new SwishPaymentSaleTransaction(msisdn);
+        }
+
+
+        public SwishPaymentSaleTransaction Transaction;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)